drivers/pds/dataservice/dataservices.go

```diff
@@ -323,6 +334,10 @@ func (d *DataserviceType) CreateSchedulerContextForPDSApps(pdsApps []*pds.Models
 	}
 	return Contexts
 }
+func init() {
+	err = pdsdriver.Register(driverName, &DataserviceType{})
+	log.FailOnError(err, "Error while Registering pds dataservice type driver")
```

Comment:
init will before any code execution.so in case of err it will fail but will not be pushed to dashboard as testset will not be created by then.
Please check once if it is fine

Reply:
yeah makes sense, I have followed like other init methods and just logging the error